manager/tasks.py

```diff
@@ -330,37 +331,46 @@ def update_kg(self, question_id, user_email=None):
 
 @celery.task(bind=True, name='pubmed', exchange='manager', routing_key='manager.pubmed', task_acks_late=True, track_started=True, worker_prefetch_multiplier=1, rate_limit='1/s')
 def fetch_pubmed_info(self, pmid, pubmed_cache_key):
-    # Actually make the request
-    postUrl = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi"
-    postData = {"db": "pubmed", "id": pmid, "version": "2.0", "retmode": "json"}
-
-    response = requests.post(postUrl, data=postData)
-
-    # When you get the request back
-    if not response.ok:
-        # logger.debug(f'Pubmed returned a bad response for {pmid}, status code {response.status_code}')
-        if response.status_code == 429:
-            return 'Too many robokop pubmed requests'
```

Comment:
Geez, we return error messages as strings instead of raising errors... Some day soon this code will be deprecated and we can forget our past mistakes.